### ISSUE TITLE
[Icon Request] Support for geojson

### ISSUE THREAD

**Issue by Dzxwind** [vscode-icons/vscode-icons#2682]
- [ ] I'm sure this issue is not a _duplicate_.
- [x] I'm submitting an icon request:
  - Type: `filename` or `foldername`
  - Icon Name: `geojson`
  - Filenames: `.geojson` or `geojson Geojson GeoJSON`

- Language Extensions:
- More info:
just about GIS data file

**Comment by remcohaszing** [comment]
I would like to see this being supported too. As far as I can tell, there’s not really an official icon. https://geojson.org doesn’t have an icon. Also I can’t find anything on https://en.wikipedia.org/wiki/GeoJSON. There’s https://geojson.io, but I don’t think it’s official and I’m not really a fan of its favicon. Also it’s PNG.

To me, the first idea that comes to mind is an abstract globe, pretty much what we already use for source maps. Other ideas are a map pin like [found on Google](https://www.google.com/search?q=map+pin&udm=2) or the [Leaflet](https://leafletjs.com) icon.

**Comment by Dzxwind** [comment]
https://www.svgrepo.com/download/399418/geojson-file.svg

This is not an official icon, but I have also seen this icon in many places before.
I hope this can be of help to you.

**Comment by remcohaszing** [comment]
Yes, that’s pretty much the same icon as we use for source maps :)

![](https://raw.githubusercontent.com/vscode-icons/vscode-icons/0b949844ed457a1032ef11e63bbb3d6b31c84262/icons/file_type_jsmap.svg)
